src/transports/ipc.js

```diff
@@ -17,6 +18,13 @@ function getIPCPath(id) {
   if (process.platform === 'win32') {
     return `\\\\?\\pipe\\discord-ipc-${id}`;
   }
+  if (process.platform === 'darwin') {
+    let tmpdir = os.tmpdir();
+    if (!tmpdir.endsWith('/T')) {
+      tmpdir += '/..';
```

Comment:
Alternatively, you can also do `tmpdir = path.dirname(tmpdir);` ([docs](https://nodejs.org/dist/latest-v15.x/docs/api/path.html#path_path_dirname_path)) to make it go back to its containing directory.

Reply:
Ah yeah that is nicer. Will update accordingly.